gulp/watch.js

```diff
@@ -113,7 +113,7 @@ module.exports = function (gulp, gutil) {
     // TODO: currently goes through all files -
     // try optimizing this for processing updated file only
     watch()(watchSrc(textPropPattern), watchOpts, function (file) {
-      var dirPath = '../dist/plugin/res/lang/feature' || 'plugin/res/lang/feature';
+      var dirPath = 'plugin/res/lang/feature';
```

Comment:
This is the "dest" path - where the files get written out to as part of this watcher. Should this be something like:

```
var dirPath = gutil.env.newStructure ? 'dist/plugin/res/lang/feature' : 'plugin/res/lang/feature';
```

To test this you'd need to make a change to a text file and see if this watch a.) is correctly being hit, b.) that it's correctly copying the changed text file into the plugin artifact (in dist).

We should verify all watchers work as they did before.